packages/env/src/internal/multi-model/node.ts

```diff
@@ -7,6 +7,13 @@ export {
 } from "./shared.js";
 
 export async function loadTransformers(onLoad: OnLoad) {
+  const nodeVersions = process.versions.node.split(".");
```

Comment:
this is not working, npm install will throw error before you do this

Reply:
@himself65 hmm, on ci we're using pnpm and it works. what do you suggest to check for the node version at runtime instead?

Reply:
npm wont work